spec/factories/factories.rb

```diff
@@ -0,0 +1,93 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:first_name) { |n| "FirstName#{n}" }
+    sequence(:last_name)  { |n| "LastName#{n}" }
+    sequence(:email)      { |n| "email-#{n}@example.com" }
+    confirmed_at { Time.now }
+    password 'password'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.